haystack/pipelines/standard_pipelines.py

```diff
@@ -757,3 +758,25 @@ def run_batch(  # type: ignore
         :param index: Optionally specify the name of index to query the document from. If None, the DocumentStore's default index (self.index) will be used.
         """
         return self.run(document_ids=document_ids, filters=filters, top_k=top_k, index=index)
+
+
+class TextIndexingPipeline(BaseStandardPipeline):
+    def __init__(self, document_store):
+        """
+        Initialize a basic Pipeline that converts text files into Documents and indexes them into a DocumentStore.
+
+        :param document_store: The DocumentStore to index the Documents into.
+        """
+
+        self.pipeline = Pipeline()
+        text_converter = TextConverter()
+        preprocessor = PreProcessor()
+        self.pipeline.add_node(component=text_converter, name="TextConverter", inputs=["File"])
+        self.pipeline.add_node(component=preprocessor, name="PreProcessor", inputs=["TextConverter"])
+        self.pipeline.add_node(component=document_store, name="DocumentStore", inputs=["PreProcessor"])
```

Comment:
I would let users provide these two instances as optional __init__ parameters, so if they want to tweak them, give them some parameters, etc... they have a way.

I'd also save their instances as attributes so in case a user want to access such objects later (for any reason) that would be possible.

So to summarize:
```suggestion
    def __init__(self, document_store: BaseDocumentStore, text_converter: Optional[TextConverter] = None, preprocessor: Optional[PreProcessor] = None):
        """
        Initialize a basic Pipeline that converts text files into Documents and indexes them into a DocumentStore.

        :param document_store: The DocumentStore to index the Documents into.
        """

        self.pipeline = Pipeline()
        self.document_store = document_store
        self.text_converter = text_converter or TextConverter()
        self.preprocessor = preprocessor or PreProcessor()
        self.pipeline.add_node(component=self.text_converter, name="TextConverter", inputs=["File"])
        self.pipeline.add_node(component=self.preprocessor, name="PreProcessor", inputs=["TextConverter"])
        self.pipeline.add_node(component=self.document_store, name="DocumentStore", inputs=["PreProcessor"])
```